homeassistant/components/tesla_fleet/sensor.py

```diff
@@ -439,6 +491,12 @@ async def async_setup_entry(
                 for description in ENERGY_LIVE_DESCRIPTIONS
                 if description.key in energysite.live_coordinator.data
             ),
+            (  # Add energy site past
+                TeslaFleetEnergyPastSensorEntity(energysite, description)
+                for energysite in entry.runtime_data.energysites
+                for description in ENERGY_PAST_DESCRIPTIONS
+                if description.key in energysite.past_coordinator.data
```

Comment:
```suggestion
                if energysite.info_coordinator.data.get("components_battery")
                or energysite.info_coordinator.data.get("components_solar")
```

Reply:
Users with only a powerwall cant use this API so no point even trying to add it.